ctpolicy/ctpolicy.go

```diff
@@ -125,36 +113,68 @@ func (ctp *CTPolicy) GetSCTs(ctx context.Context, cert core.CertDER, expiration
 	// cert's expiry. Randomize the order of the logs so that we're not always
 	// trying to submit to the same two.
 	logs := ctp.sctLogs.ForTime(expiration).Permute()
+	candidateLogs := len(logs)
 
-	// Kick off a collection of goroutines to try to submit the precert to each
-	// log. Ensure that the results channel has a buffer equal to the number of
+	// Ensure that the results channel has a buffer equal to the number of
 	// goroutines we're kicking off, so that they're all guaranteed to be able to
 	// write to it and exit without blocking and leaking.
-	resChan := make(chan result, len(logs))
-	for i, log := range logs {
-		go ctp.getOne(subCtx, cert, i, log, resChan)
+	resChan := make(chan result, candidateLogs)
+
+	if candidateLogs < 2 {
+		return nil, berrors.MissingSCTsError("Insufficient CT logs available (%d)", candidateLogs)
+	}
```

Comment:
nit: this should be moved up, to be immediately below the initialization of `candidateLogs`. There's no reason to break up that init/check with the creation of `resChan`.

Reply:
Sure, happy to do it this way